templates/commands/render.go

```diff
@@ -367,76 +366,123 @@ func parseAndExecuteGoTmpl(m model.String, inputs map[string]string) (string, er
 }
 
 // "from" may be a file or directory. "pos" is only used for error messages.
-func copyRecursive(pos *model.ConfigPos, from, to string, rfs renderFS, overwrite bool) error {
-	return fs.WalkDir(rfs, from, func(path string, d fs.DirEntry, err error) error { //nolint:wrapcheck
+func copyRecursive(pos *model.ConfigPos, fromRoot, toRoot string, rfs renderFS, overwrite, dryRun bool) error {
```

Comment:
Another nit: sometimes we use `from` and `to` and sometimes we use `dst`. Does it make sense to consistently use `src` and `dst`? Like `srcRoot` and `dstRoot`. This is also consistent with the parameters naming in io.Copy

Reply:
I like `src` and `dst`

Reply:
Updated with src and dst.